clients/src/test/java/org/apache/kafka/common/utils/Serializer.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements. See the NOTICE
+ * file distributed with this work for additional information regarding copyright ownership. The ASF licenses this file
+ * to You under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package org.apache.kafka.common.utils;
+
+import java.io.IOException;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.ObjectInputStream;
+
+/**
+ * Created by praveendevarao on 14/01/16.
+ */
+public class Serializer {
+
+    public void serialize(Object toSerialize) throws IOException {
+        ByteArrayOutputStream arrayOutputStream = new ByteArrayOutputStream();
+        ObjectOutputStream ooStream = new ObjectOutputStream(arrayOutputStream);
+        ooStream.writeObject(toSerialize);
+        arrayOutputStream.close();
+        ooStream.close();
+    }
+
+    public Object deSerialize(String fileName) throws IOException, ClassNotFoundException {
```

Comment:
`deserialize` is a single word, that's why it's not needed. See https://www.google.co.uk/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=deserialize